dependencies.dot

```diff
@@ -0,0 +1,18 @@
+digraph {
+  Pace [shape=box]
+  fv3core [shape=oval]
+  driver [shape=oval]
+  physics [shape=oval]
+  util [shape=oval]
+  
+  Pace -> fv3core
+  Pace -> physics
+  Pace -> util
+  Pace -> driver
+  driver -> fv3core
+  driver -> physics
+  driver -> util
+  fv3core -> util
+  physics -> util
```

Comment:
is there still a plan for a separate 'shared' directory? or are you leaning towards incorporating those items into util?